tests/Integration/Execution/MutationExecutor/MorphManyTest.php

```diff
@@ -439,6 +439,50 @@ public function testUpdateAndDeleteMorphMany(string $action): void
         ]);
     }
 
+    /**
+     * @dataProvider existingModelMutations
+     */
+    public function testShouldNotDeleteWithoutRelationUpdateAndDeleteMorphMany(string $action): void
+    {
+        $images = [factory(Image::class)->create(), factory(Image::class)->create()];
+        factory(Task::class)
+            ->createMany([[], []])
+            ->each(function (Task $task, int $index) use ($images): void {
+                $task->images()->save($images[$index]);
+            });
+
+        $this->graphQL(/** @lang GraphQL */ "
+        mutation {
+            {$action}Task(input: {
+                id: 1
+                name: \"foo\"
+                images: {
+                    delete: [1]
```

Comment:
Which ID is that?

Reply:
The IDs are now referenced from the created entities. It is still implicit because I do not explicitly set the IDs on creation but other tests are also hardcoding the IDs into the query so I guess it's okayyyy.